cloudevents/bindings/kafka-protocol-binding.md

```diff
@@ -182,6 +182,11 @@ directly to the CloudEvents `datacontenttype` attribute.
 The [`data`](#21-data) byte-sequence MUST be used as the value of the Kafka
 message.
 
+A CloudEvent with no `data` can be represented in a Kafka message in _binary_ mode,
```

Comment:
Is this saying that in order to send an event w/o any data it MUST be in binary mode, or are you just saying that it's legal to send a binary CE w/o a `data` value? It could be read both ways. I think you mean the latter, correct?

Reply:
Yes, the latter. Happy to clarify it. An event w/o data in structured mode is fine too, but *doesn't* represent a tombstone.

Reply:
yup - that was my assumption - and it might be good to mention that tombstone doesn't apply for structured mode. Just to make it painfully clear

Reply:
Cool. Will amend the PR tomorrow; no need to talk about it today unless we think we'd get other and different feedback.